### PR TITLE
Modifying alerting logic based on alert classifications

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bugid: Tendrl/monitoring-integration#218
tendrl-bugid: Tendrl/monitoring-integration#216

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

File: tendrl/monitoring_integration/grafana/webhook_receiver.py [hunk 1]
```diff
@@ -20,22 +21,20 @@ def __init__(self):
...
                 )
-                start_response(
-                    '200 OK',
-                    [('Content-Type', 'text/html')]
-                )
-                response = [b'<h1>Alert Received</h1>']
+                response = Response('Alert received successfully')
+                response.headers['content-length'] = len(response.data)
+                response.status_code = 200
         except (IOError, AssertionError) as ex:
```
Comment (rishubhjain) [inline tendrl/monitoring_integration/grafana/webhook_receiver.py]: is AssertionError required?

File: tendrl/monitoring_integration/grafana/webhook_receiver.py [hunk 2]
```diff
@@ -20,22 +21,20 @@ def __init__(self):
...
+                response.headers['content-length'] = len(response.data)
+                response.status_code = 404
             else:
-                data = env['wsgi.input'].read()
+                data = env['wsgi.input'].read(
+                    int(env['CONTENT_LENGTH'])
+                )
                 data = json.loads(data)
                 self.alert_handler.handle_alert(
                     data["ruleId"]
```
Comment (rishubhjain) [inline tendrl/monitoring_integration/grafana/webhook_receiver.py]: Can we put a check for ruleId (For edge cases), or the ruleId will be there no matter what?

File: tendrl/monitoring_integration/grafana/webhook_receiver.py (hunk 1 above)
Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/grafana/webhook_receiver.py]: yes it required

File: tendrl/monitoring_integration/grafana/webhook_receiver.py (hunk 2 above)
Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/grafana/webhook_receiver.py]: ok i will do it

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n @nnDarshan @rishubhjain please review

**Comment by GowthamShanmugam** [comment]
Tendrl/gluster-integration#449

**Comment by GowthamShanmugam** [comment]
@r0h4n verified

**Pull request merged**